functions/mqhandlers.go

```diff
@@ -200,7 +200,6 @@ func HostPeerUpdate(client mqtt.Client, msg mqtt.Message) {
 			}
 		}
 	}
-	_ = UpdateHostSettings()
```

Comment:
why remove this?

Reply:
not required, this happens during check-in already